### PR TITLE
Feature/OB-29

### PR DESCRIPTION
**Pull request by csabbee**

### CODE REVIEW

File: backend/online-bookmark/api/swagger/swagger.yaml [hunk 1]
```diff
@@ -226,6 +316,13 @@ definitions:
           type: string
       url:
         type: string
+  User:
+    required: ["email", "password"]
```
Comment (ppseprus) [inline backend/online-bookmark/api/swagger/swagger.yaml]: `hash` would be less missleading...

File: backend/online-bookmark/api/swagger/swagger.yaml [hunk 2]
```diff
@@ -226,6 +316,13 @@ definitions:
...
+    properties:
+      email:
+        type: string
+      password:
```
Comment (ppseprus) [inline backend/online-bookmark/api/swagger/swagger.yaml]: Here too...

File: backend/online-bookmark/api/swagger/swagger.yaml [hunk 3]
```diff
@@ -192,7 +228,61 @@ paths:
...
+        - name: user
+          in: body
+          required: true
+          schema:
+            $ref: "#/definitions/User"
+      responses:
+        "200":
+          description: User created
+        "409":
+          description: Username, or email address in use
```
Comment (ppseprus) [inline backend/online-bookmark/api/swagger/swagger.yaml:251]: You say, the username or the email address is in use, yet I don't see "username" among the required fields. Am I not seeing something?

File: backend/online-bookmark/api/swagger/swagger.yaml (hunk 1 above)
Comment (ppseprus) [inline backend/online-bookmark/api/swagger/swagger.yaml]: No "username" among the required fields...

Comment (csabbee) [reply backend/online-bookmark/api/swagger/swagger.yaml]: Why would it be? You have to send your password.

File: backend/online-bookmark/api/swagger/swagger.yaml (hunk 3 above)
Comment (csabbee) [reply backend/online-bookmark/api/swagger/swagger.yaml:251]: Nope, left it out by mistake. Thanks:)

File: backend/online-bookmark/api/swagger/swagger.yaml (hunk 1 above)
Comment (csabbee) [reply backend/online-bookmark/api/swagger/swagger.yaml]: +1

Comment (ppseprus) [reply backend/online-bookmark/api/swagger/swagger.yaml]: So.. Why?

**Review by ppseprus — APPROVED** [review]

**Pull request merged**